### ISSUE TITLE
Search not returning any results

### ISSUE THREAD

**Issue by antigravity83** [Jackett/Jackett#1755]
Jackett version: 0.8.128.0

As per title, not returning any results, via Manual or Sonarr/Radarr:

Here's logs of Manual Search:
https://zerobin.net/?313ed12bd971eb2c#PkteiYFgyY582DLgF3K+2IkNSzArx2Hj5m0VaDXwsIs=

**Comment by antigravity83** [comment]
Looks to be related to XSpeeds and Immortalseed not working.

When I isolate a different tracker to search with- results are returned.

**Comment by antigravity83** [comment]
Kaso17, why am I not recieving any results from any tracker when these two are the only ones down?

Shouldn't i still recieve results from my other active trackers?

**Comment by kaso17** [comment]
yes you should, please restart jackett with the -t -l options and post the resulting log of a manual search.
